src/components/overlays/Popup/Popup.tsx

```diff
@@ -98,6 +106,7 @@ export default class Popup extends React.Component<IProps, IState> {
 						[styles.Popup__PositionBottom]: this.props.position === 'bottom',
 						[styles.Popup__PositionRight]: this.props.position === 'right',
 						[styles.Popup__PositionTop]: this.props.position === 'top',
+						[styles.Popup__CenteredTail]: this.props.centerTail,
```

Comment:
@flycrum I don't love the naming of this prop, `centerTail`, or the inflexibility of it. Ideally, this prop would be able to take multiple options such as `left`, `center` and `right` but this becomes problematic to manage/reason about when you take into account that the tail can be positioned on either the `bottom`, `right` or `top`. An example would be, what does a tail position of `left` mean if the tail is also on the left side of the `Popup`?

This general thought process is what led me to simplify to the current prop, `centerTail`. Do you have any ideas to clean this up or make it more flexible?

Also note that this prop has not been tested with instances where the `tail` is positioned on the top or left side of the `Popup`